pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -153,10 +166,14 @@ type vpnSeedServer struct {
 	podNetwork               string
 	nodeNetwork              *string
 	replicas                 int32
+	highAvailabilityEnabled  bool
+	highAvailabilityServers  int
+	highAvailabilityClients  int
 	istioIngressGateway      IstioIngressGateway
 	exposureClassHandlerName *string
 	sniConfig                *config.SNI
 	secrets                  Secrets
+	seedVersion              string
```

Comment:
Same like in `vpnshoot` package

Reply:
changed plus refactoring using a Values struct